scripts/exregional_nexus_emission.sh

```diff
@@ -89,7 +89,7 @@ mkdir -p "$DATAinput"
 USE_GFS_SFC="FALSE"
 
 if [ "${RUN_ENVIR}" = "nco" ]; then
-  GFS_SFC_INPUT="${DATAROOT}/nexus_gfs_sfc.${share_pid}"
+  GFS_SFC_INPUT=${GFS_SFC_INPUT:-${DATAROOT}/aqm_nexus_gfs_sfc_${cyc}.${share_pid}}
```

Comment:
Same as above

Reply:
The model name need to be included in DATA directory naming logic.